### PR TITLE
Update README and InputHistory in UserGuide

### PR DESCRIPTION
**Pull request by garyljj**
Update README.md to PartyPlanet
Added InputHistory function details to UserGuide.md
Mainly:
- Up/down arrow cycles through history
- Esc clears input

Part of #93

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -159,6 +159,17 @@ Exits the app.
 
 Format: `exit`
 
+### InputHistory / Keyboard shortcuts :
+
+Retrieves previously entered input.
+
+* Entering new command adds new entry to InputHistory.
+* InputHistory will save last 20 inputs.
+* Pressing `Up` arrow key in the text input panel reverts to eariler input.
```
Comment (zhengruoxin) [inline docs/UserGuide.md:168]:
```suggestion
* Pressing `Up` arrow key in the text input panel reverts to earlier input.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -159,6 +159,17 @@ Exits the app.
...
+* Pressing `Down` arrow key undoes the history revert.
+* At the most recent input, `Down` arrow key once more clears the text box.
```
Comment (zhengruoxin) [inline docs/UserGuide.md:170]:
```suggestion
* At the most recent input, pressing `Down` arrow key once more clears the text box.
```

File: docs/UserGuide.md (hunk 1 above)
Comment (zhengruoxin) [inline docs/UserGuide.md:168]:
```suggestion
* Pressing `Up` arrow key in the text input panel reverts to earlier input.
```

**Review by zhengruoxin — CHANGES_REQUESTED** [review]
Just a typo, LGTM!

**Review by pyuxiang — APPROVED** [review]
LGTM

**Review by zhengruoxin — APPROVED** [review]

**Pull request merged**